src/AddTodoForm.jsx

```diff
@@ -0,0 +1,11 @@
+export default function AddTodoForm() {
+  return (
+    <form>
+      <label htmlFor="todoTitle">
+        Title
+      </label>
+      <input id="todoTitle"placeholder="Type in todo to add to the list"/>
```

Comment:
Nice usage of placeholder text here! This kind of consideration for user-friendly design is important in production-level software.

Reply:
Thank you for the review!